templates/graviteeio-apim/0/rancher-compose.yml

```diff
@@ -0,0 +1,69 @@
+.catalog:
+  name: Gravitee.io APIM
+  version: v1-rancher
+  description: Gravitee.io API Management
+  uuid: graviteeio-0
+  questions:
+    - variable: "docker_version"
+      description: "Gravitee.io version"
+      label: "Gravitee.io version"
+      required: true
+      default: "latest"
+      type: "string"   
+    - variable: "gateway_port"
+      description: "Public gateway port"
+      label: "Public gateway port"
+      required: true
+      default: "8000"
+      type: "int"
+    - variable: "ui_port"
+      description: "Public Management UI port"
+      label: "Public Management UI port"
+      required: true
+      default: "8002"      
+      type: "int" 
+    - variable: "management_port"
+      description: "Public Management API port"
+      label: "Public Management API port"
+      required: true
+      default: "8001"      
+      type: "int"              
+    - variable: "management_host"
+      description: "Public Management API Hostname or IP (Your rancher host node)"
+      label: "Public Management API Host"
+      required: true
+      default: ""
+      type: "string"
+
```

Comment:
Add service combo questions to select elasticsearch and mongodb services. You need to deploy them before.
    - variable: "elasticsearch_service"
      description: |
        Choose the elasticsearch service to use. 
      label: "Elasticsearch stack/service"
      default: "elasticsearch-2/kopf"
      required: true
      type: "service"
    - variable: "mongodb_service"
      description: |
        Choose the mongodb service to use. 
      label: "Mongodb stack/service"
      default: "mongoDB/mongo-cluster"
      required: true
      type: "service"